### ISSUE TITLE
AR_X_0.83.5

### ISSUE THREAD

**Issue by zakhad** [bob0bob/ARX#24]
Picking up any potion instantly immobilises you, does it have an incorrect weight assigned?
![image](https://github.com/bob0bob/ARX/assets/5264667/f0769055-c4eb-40d1-a40c-e69fafd4e74e)
![image](https://github.com/bob0bob/ARX/assets/5264667/4faae795-0e82-45d9-b46b-1eceea31a7c2)
![image](https://github.com/bob0bob/ARX/assets/5264667/85fa3a3e-4187-48b3-b8eb-846ef4263837)

**Comment by bob0bob** [comment]
Are you using the Java version of the C version.   The C version has been abandoned and I moved it over to Java's LWJGL3 opengl library.

**Comment by zakhad** [comment]
this is where i got it from, and the version i am playing with i have no idea if it's c or java. i presume it's the C version but the website doesn't have url for java version it just says coming soon 2018?

https://www.landbeyond.net/arx/index.php/downloads

The only other website that I haven't used in a few years as it looks abandoned.

https://www.crpgdev.com/downloads.html

**Comment by bob0bob** [comment]
Okay,  that is the "C" version.  Let me get you a copy of the Java version,  it is about 90% code complete with tons of enhancements that can be turned on/off.

**Comment by zakhad** [comment]
I was getting frustrated that i couldn't buy any city armour in town, no matter what part of the set i was trying to equip it kept equipping to the head slot lol, like city armour was different than dungeon armour.

But sure i would love to try out the java version and give feedback or help you test as you need, i grew up playing this game.

**Comment by ltdanman** [comment]
Thank you for helping !!  I can't wait to play this game!!!
Dan

Sent from Yahoo Mail on Android 
 
  On Thu, Nov 2, 2023 at 12:33 PM, ***@***.***> wrote:   


I was getting frustrated that i couldn't buy any city armour in town, no matter what part of the set i was trying to equip it kept equipping to the head slot lol, like city armour was different than dungeon armour.

But sure i would love to try out the java version and give feedback or help you test as you need, i grew up playing this game.

**Comment by bob0bob** [comment]
Send me an email at   info  at landbeyond.net   and I'll get you links and beta code for testing out the product.

**Comment by zakhad** [comment]
@bob0bob did you get my email?

**Comment by zakhad** [comment]
![image](https://github.com/bob0bob/ARX/assets/5264667/1d4a9f8b-30c6-4fb5-9adf-9d157679829b)

oddly enough that's only just shown up

**Comment by ltdanman** [comment]
i did not get any email

________________________________

**Comment by zakhad** [comment]
![image](https://github.com/bob0bob/ARX/assets/5264667/515eed40-7d90-49c6-961e-265d94afe1a2)

what happened to me when i emailed him

**Comment by ltdanman** [comment]
land beyond email is down
    On Friday, November 3, 2023 at 02:17:22 PM EDT, Zakhad ***@***.***> wrote:  
 
 




what happened to me when i emailed him

**Comment by zakhad** [comment]
![image](https://github.com/bob0bob/ARX/assets/5264667/cc91f732-c6cf-4dde-8a2f-3a7007e8d14b)

Still getting email failed. @bob0bob

**Comment by jgrochoski** [comment]
Is the beta still ongoing?  I lost access years ago (the beta tab and forums have been down for years), but would love to get re-engaged.

**Comment by jgrochoski** [comment]
Also curious if it's coming to steam soon?  I *found* an app while interrogating steam APIs with id 1138110 and name Alternative_Reality_Demo
It's unlisted, but technically exists in their library

**Comment by zakhad** [comment]
He fixed the email and i asked him about it but have yet to get a reply other than thanking me for informing him about the broken email.

**Comment by ltdanman** [comment]
email fixed great!  is the beta still being developed?
________________________________

**Comment by carlSankey** [comment]
Great to here that you have fixed your email. I have created a new branch and  starting fixing some of the issues within the C++ version, including the immobilization bug.  I've added most spells, including light, supervision and some combat spells (although I'm not that happy with the amount of damage done by the spells) added fatigue and fixed a few issues with the combat.

**Comment by bob0bob** [comment]
I'm not working on the C version of the game.  I converted it to Java and it is code complete.  It is  truly remake.  I spent 6+ months dissembling the code and making sure the Java version is absolutely in sync with the original "Dungeon" code when it comes to everything.   Combat, movement, spells and etc....

The Java version has enhancement that you can turn on, but the core is still the same.  It supports 3d Models,  Homes,  more random encounters.  Achievements, Side Quests, certain areas of the city has more common encounters for that area.  Higher crime areas will get more bad encounter vs. good area.

There is a church in the City now, it is the same as the Dungeon Church.

People live in homes, and have a life and schedule.   

Side Quests can be complete or you could just do part of it and keep what they want, it is up to you. 
Guard Towers.

Spells, are accurate and complete to the original game and guilds are complete also.

**Comment by bob0bob** [comment]
Yes,  that is why I said you could continue with the C if you want.  I'm just starting to get back into the development for the game.  last couple of years has had a huge change in life and family.   From the pandemic to moving to a different state, and relocating my business on top of that,  it has been a full time job.

Now it is training new employee to get back up to par in the company in processing and handling orders all over again with new people.

**Comment by ltdanman** [comment]
where can we download and play the completed java version?
________________________________

**Comment by bob0bob** [comment]
Give me a week to get things together.

**Comment by Larsone93** [comment]
Hi, there. I recently started playing this in a wave nostalgia. I noted a couple of bugs already cited here and was going to ask some questions about building this in a Linux environment.

But then you mentioned a Java version. 

Is the Beta open that I could try it out?

**Comment by bob0bob** [comment]
Yes, it is open to try it out.   Give me about a week and I'll send you a link to get it.

**Comment by bob0bob** [comment]
In the mean time, have you checked out [https://www.youtube.com/@LandBeyond](https://www.youtube.com/@LandBeyond) my youtube channel showing progress of the development.

**Comment by carlSankey** [comment]
I have updated my version of this project which has a few fixes. [https://github.com/carlSankey/ARX] The release folder has the executable.

**Comment by Larsone93** [comment]
I'll try it out this weekend. 

I recently got this code base to build in Linux using mingw-w64 and I tried my hand at fixing a few things. It will be fun to check it out against your updates to see how close I got.
